tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -78,9 +77,18 @@ sudo snap install --classic snapcraft
 
 positive
 : **NOTE:**
-    * If the `snap` command is not available, install the `snapd` package via APT.
     * The `--classic` switch enables the installation of a snap that uses *classic confinement*.  We discuss snap security confinement in the following section.
 
+Next we will install multipass to manage the Virtual Machines in which snapcraft will build.
```

Comment:
I don't think this section is required anymore.  It looks like multipass is shipping with the latest stable snapcraft.